test/unit/org/apache/cassandra/net/ForwardingInfoTest.java

```diff
@@ -41,12 +41,6 @@ public void testCurrent() throws Exception
         testVersion(MessagingService.current_version);
     }
 
-    @Test
-    public void test30() throws Exception
```

Comment:
Now `ForwardingInfoTest#testVersion(int)` is always called with the same value. I think it can be either inlined into `ForwardingInfoTest#testCurrent()` or annotated with `@SuppressWarnings("SameParameterValue")`.

Reply:
going to loop through supportedVersions()